recipes/multineat/meta.yaml

```diff
@@ -0,0 +1,58 @@
+
+{% set name = "multineat" %}
+{% set version = "0.5.1" %}
+{% set sha256 = "d58439c548433adcda98e695be53e526ba940a4b9c44fb9a05d92cd495cdd47f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_tag: release/{{ version }}
+  git_url: https://github.com/MultiNEAT/MultiNEAT.git
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - cmake
+    - psutil
+    - boost=1.66*
+    - boost-cpp=1.66*
+  run:
+    - python
+    - psutil
+    - boost=1.66*
+    - boost-cpp=1.66*
```

Comment:
Do you really need a runtime pin here?

Reply:
Updated to min requirement of >=1.65

Reply:
Seems to be not working... 1.67 has different naming convention for boost-python. So only 1.65 and 1.66 are compatible at the moment.
Is there a way to specify version range?